### DISCUSSION TITLE
Issue in viewing image

### DISCUSSION THREAD

**Discussion by himanshu-negi45** [requarks/wiki#3529] [Error / Bug Report]
Hi Team,

I have recently migrated my wiki from V1 to V2 May release. now all function seems working fine but while i am opening some files the image previews are not showing thought the document is visible but the image attached in that is not cant be seen. Kindly suggest.

![image](https://user-images.githubusercontent.com/66419482/84037874-d1fa2980-a9bc-11ea-8763-c8f2dd79b0ab.png)
@team

**Comment by HerbertNagy** [comment]
Resolved this situation? if yes, please give us the answer
